great_expectations/data_context/store/inline_store_backend.py

```diff
@@ -1,6 +1,10 @@
 from typing import Any, List, Optional, Tuple
 
 from great_expectations.data_context.store.store_backend import StoreBackend
+from great_expectations.data_context.types.base import DataContextConfig
+from great_expectations.data_context.types.data_context_variables import (
+    DataContextVariableSchema,
+)
 from great_expectations.exceptions.exceptions import StoreBackendError
 from great_expectations.util import filter_properties_dict
 
```

Comment:
I know this code wasn't modified in this PR, and can be done in a different PR but we should have a docstring for the class that describes what an InlineStoreBackend is. Including mentioning that it is only for use with file-backed data contexts (`DataContext` and `FileDataContext`).